wrapped.js

```diff
@@ -49,6 +59,18 @@ function WrappedError(options) {
             enumerable: false
         });
 
+        var causeType = err.cause.type;
+        if (!causeType && nodeCause) {
+            causeType = 'error.wrapped-io.' +
+                (err.syscall || 'unknown') + '.' +
+                (err.errno || 'unknown');
+        } else {
+            causeType = 'error.wrapped-unknown';
+        }
+
+        err.fullType = err.type + '~' +
+            (err.cause.type || causeType);
```

Comment:
`-` and `~` are difficult to distinguish visually. You might consider something more distinct.